src/web/mjs/connectors/UnionMangas.mjs

```diff
@@ -44,11 +53,12 @@ export default class UnionMangas extends Connector {
                 title: element.text.trim()
             };
         });
-    }
 
+    }
     async _getPages(chapter) {
         const uri = new URL(chapter.id, this.url);
         const request = new Request(uri, this.requestOptions);
+        request.headers.set('x-cookie', `modoLeitura=2; path=${manga.id}` ); //in case cookie is needed to get all pictures
```

Comment:
```suggestion
        request.headers.set('x-cookie', `modoLeitura=2; path=${chapter.manga.id}` ); //in case cookie is needed to get all pictures
```

Reply:
oops !